lib/config/config-utils.go

```diff
@@ -165,8 +165,13 @@ func (c *Configuration) loadIcon() *errco.MshLog {
 // In case of error "", -1, *errco.MshLog are returned.
 //
 // (checkout version.json info: https://minecraft.fandom.com/wiki/Version.json)
-func (c *Configuration) getVersionInfo() (string, int, *errco.MshLog) {
-	reader, err := zip.OpenReader(filepath.Join(c.Server.Folder, c.Server.FileName))
+func (c *Configuration) getVersionInfo(altServerName ...string) (string, int, *errco.MshLog) {
```

Comment:
@JackGlobetrotter why `altServerName ...string` ? isn't it enough to have one string parameter?

Reply:
@gekigek99 i wanted to make the parameter optional, so that the function could be used w/o params if needed elswhere in the futur? But could also be `altServerName string` if not needed